src/Workspaces/Remote/ServiceHub/Services/CodeAnalysisService_SymbolFinder.cs

```diff
@@ -15,9 +15,9 @@ namespace Microsoft.CodeAnalysis.Remote
     // root level service for all Roslyn services
     internal partial class CodeAnalysisService : IRemoteSymbolFinder
     {
-        public Task FindReferencesAsync(SerializableSymbolAndProjectId symbolAndProjectIdArg, DocumentId[] documentArgs, CancellationToken cancellationToken)
+        public Task FindReferencesAsync(SerializableSymbolAndProjectId symbolAndProjectIdArg, DocumentId[] documentArgs, CancellationToken givenCancellationToken)
```

Comment:
❕ The public signatures should use the common form for this parameter (always named `cancellationToken`). Use a different name for the inner anonymous function instead. (Applies to all signatures in the internal API.)

Reply:
sure

Reply:
I am doing it. but using name "cancellation" for one that should never be actually used is more confusing.